demo_serial/src/main/java/com/novoda/downloadmanager/demo/serial/MainActivity.java

```diff
@@ -52,8 +54,11 @@ private void setupDownloadingExample() {
                 new View.OnClickListener() {
                     @Override
                     public void onClick(View v) {
-                        long id = downloadManager.enqueue(request);
-                        Log.d(TAG, "Download starting with id: " + id);
+                        long batchId = downloadManager.create(batch);
+                        request.setBatchId(batchId);
```

Comment:
This will ultimately change. As discussed with @blundell as well we will have an API that will enable to enqueue a batch directly as opposed to separate requests. We've chosen not to include this into this PR as the diff would have got even more massive.